django_loci/geo/settings.py

```diff
@@ -0,0 +1,21 @@
+from django.conf import settings
+
+DJANGO_LOCI_GEOCODER = getattr(settings,
+                               'DJANGO_LOCI_GEOCODER',
+                               'ArcGIS')
+
```

Comment:
remove blank lines between each setting and write all the setting definition on one line if possible because they're pretty simple definition which do not need to be laid out on multiple lines (we do that for more complex data structures).